packages/homepage/content/docs/6-templates.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Templates'
+authors: ['Saeris']
+description: 'CodeSandbox allows you to create your  custom templates from sandboxes'
+---
+
+## What are Templates?
+
+Templates are a combination of an execution [Environment]() and a set of starter files, similar to what you would find in the output of a framework CLI such as Create React App or Gatsby CLI. CodeSandbox has a number of official Templates, such as React, Vue, Gatsby and others with which you can quickly bootstrap a new project.
+
+Users can also create their own custom Templates, whereby they can modify an existing template with their own dependencies, files and folder structure, and configuration. Once a custom Template has been created, the user can then start new projects from the Create New Sandbox modal using one of their Templates.
+
+## How to make a custom Template
+
+There are a few ways to make a Template, but the primary way to do so is from within the Project Info panel inside the Editor. When viewing one of your Sandboxes, you'll see a button at the bottom of the Project Info panel that says `Make Template`. Clicking on this will convert the Sandbox to a Template, which will freeze it and make it available inside of the Create New Sandbox modal.
```

Comment:
```suggestion
There are a few ways to make a template, but the primary way to do so is from within the "Project Info" panel inside the editor. When viewing one of your sandboxes, you'll see a button at the bottom of the "Project Info" panel that says `Make Template`. Clicking on this will convert the sandbox to a template, which will freeze it and make it available inside of the "Create New Sandbox" modal.
```